src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    difference is that the garbage collector doesn't take into account
+    `wal.cleanup_delay` if all the nodes in the replica set are up and running,
+    which may lead to the removal of the required write-ahead logs.
+]])
+
+-- }}} wal configuration
+
+-- }}} Instance descriptions
+
+-- {{{ Cluster descriptions
+
+local L = {}
+
+L[''] = 'Cluster configuration'
+
+-- {{{ conditional configuration
+
+L['conditional'] = format_text([[
+    The `conditional` section defines the configuration parts that apply to
+    instances that meet certain conditions.
+]])
+
+L['conditional.*'] = format_text([[
+    `conditional.if`: Defines a section where configuration options apply only
+    if specified conditions are met.
+
+    Conditions can include `tarantool_version` and support comparison operators
+    (`>`, `<`, `>=`, `<=`, `==`, `!=`), as well as logical operators
+    (`||`, `&&`) and parentheses for grouping.
+]])
+
+L['conditional.*.*'] = format_text([[
+    `conditional.if.*`: Defines a specific condition and the configuration
+    options applied when the condition is met.
+
+    Each condition compares the `tarantool_version` using operators
+    (`<`, `>`, `<=`, `>=`, `==`, `!=`) and supports defining labels or other
+    options that are applied only if the condition holds true.
+]])
+
+-- }}} conditional configuration
+
+-- {{{ groups configuration
+
+L['groups'] = format_text([[
+    This section provides the ability to define the full topology
+    of a Tarantool cluster.
+]])
+
+L['groups.*'] = format_text([[
+    A group of replicasets.
+
+    The following rules are applied to group names:
+
+    - The maximum number of symbols is 63.
+    - Should start with a letter.
+    - Can contain lowercase letters (a-z).
+    - Can contain digits (0-9).
+    - Can contain the following characters: -, _.
+]])
+
+L['groups.*.replicasets'] = 'Replica sets that belong to this group.'
+
+L['groups.*.replicasets.*'] = format_text([[
+    A group of instances.
+
+    Note that the rules applied to a replica set name are the same as for
+    groups. Learn more in `groups.<group_name>`.
+]])
+
+L['groups.*.replicasets.*.instances'] = format_text([[
+    Instances that belong to this replica set.
+]])
+
+L['groups.*.replicasets.*.instances.*'] = 'An instance name.'
+
+L['groups.*.replicasets.*.leader'] = format_text([[
+    A replica set leader. This option can be used to set a replica set leader
+    when manual `replication.failover` is used.
+
+    To perform controlled failover, `<replicaset_name>.leader` can be
+    temporarily removed or set to null.
+]])
+
+L['groups.*.replicasets.*.bootstrap_leader'] = format_text([[
+    A bootstrap leader for a replica set. To specify a bootstrap leader
+    manually, you need to set `replication.bootstrap_strategy` to `config`.
+]])
+
+-- }}} groups configuration
+
+-- }}} Cluster descriptions
+
+return {
+    instance_descriptions = M,
+    cluster_descriptions = L,
```

Comment:
If you want one-letter table names, `I` and `C` fit better here :)

Reply:
Replaced with `C I`